docs/runbooks/HPPSharingPoolPathWithOS.md

```diff
@@ -1,34 +1,51 @@
+<!-- Edited by Jiří Herrmann, 8 Nov 2022 -->
+
 # HPPSharingPoolPathWithOS
 
 ## Meaning
 
-The hostpath-provisioner (HPP) is used to dynamically provision hostPath volumes to provide storage for PVCs.  
-If HPP is sharing a filesystem with other critical components (kubelet / OS) its possible that HPP PVs will cause node disk pressure.
+This alert fires when the hostpath provisioner (HPP) is sharing a file system with other critical components, such as `kubelet` or the operating system (OS).
+
+HPP dynamically provisions hostpath volumes to provide storage for persistent volume claims (PVCs). In the described scenario, HPP persistent volumes might cause node disk pressure.
+
+
 ## Impact
 
-Possible degraded node operation.
+The affected node might have degraded performance and stability.
 
 ## Diagnosis
 
-- Check which HPP pool is being shared with OS by looking at DaemonSet logs
-	```bash
-	export HPP_NAMESPACE="$(kubectl get deployment -A | grep hostpath-provisioner-operator | awk '{print $1}')"
-	```
+Check which HPP pool is being shared with the OS by reviewing the DaemonSet logs.
+
+1. Configure the `HPP_NAMESPACE` environment variable:
+```bash
+$ export HPP_NAMESPACE="$(kubectl get deployment -A | grep hostpath-provisioner-operator | awk '{print $1}')"
+```
 
-- Find CSI DaemonSet pods
-	```bash
-	kubectl -n $HPP_NAMESPACE get pods | grep hostpath-provisioner-csi
-	```
+2. Find the Container Storage Interface (CSI) DaemonSet pods:
+```bash
+$ kubectl -n $HPP_NAMESPACE get pods | grep hostpath-provisioner-csi
+```
  
-- Check CSI pod logs to find out which pool & path are being shared with OS;  
-The relevant log lines will look like:  
-`I0208 15:21:03.769731       1 utils.go:221] pool (legacy, csi-data-dir/csi), shares path with OS which can lead to node disk pressure`  
-of course with your chosen pool name/pool path instead.
-    ```bash
-	kubectl -n $HPP_NAMESPACE logs <csi_daemonset_pod> -c hostpath-provisioner
-	```
+3. Check the CSI pod logs to learn which pool and path are being shared with the OS:
+```bash
+$ kubectl -n $HPP_NAMESPACE logs <csi_daemonset_pod> -c hostpath-provisioner
+```
+
+The relevant log lines look similar to the following:
+```  
+I0208 15:21:03.769731       1 utils.go:221] pool (legacy, csi-data-dir/csi), shares path with OS which can lead to node disk pressure
```

Comment:
I think "legacy, csi-data-dir/csi" are examples added by whoever wrote this runbook. Maybe we should put them in `<>` so that users understand that this is not an actual example.